### DISCUSSION TITLE
建议增加功能「macOS 下触控板/鼠标滚轮调节音量方向反转」、「触控板/鼠标滚轮调节进度条」

### DISCUSSION THREAD

**Discussion by qianbinbin** [the1812/Bilibili-Evolved#4053] [功能建议 / Ideas]
为了避免歧义，以下约定触控板/鼠标滚轮的”向上“指的是”手指向上移动“。

macOS 触控板/鼠标滚轮方向与 Windows 相反，它的交互逻辑类似 iPad，即把屏幕内容看作主体，手指向上滑动时，屏幕内容也向上移动。macOS 本身也有设置项，看来苹果知道有人不习惯，不过这是系统级的调整。

目前触控板/鼠标滚轮向上时，音量都是减小的，不符合用户直觉。

作为参考：

- 自带的 QuickTime Player、第三方的 IINA：触控板/鼠标滚轮向上，对应音量上升；如果光标在进度条上，则快进（进度条右移）。
- QQ 音乐：向上对应音量上升。
- 自带的 Apple Music：触控板表现与 QuickTime 一致（但奇怪的是鼠标滚轮相反，个人觉得是 bug，不过我的是老掉牙的 Catalina，可能新版不一样）。

只有 b 站的音量调节是完全相反的，真的很反直觉，而 up 主们视频音量大小不一，经常需要调节，反转这个功能可以解决一大痛点。如果直接修改交互逻辑不太好的话，可以做成单独的设置。

至于调节进度条，这个对于广大用户应该都很有用，供参考。

谢谢！

**Comment by lovelyCARDINAL** [comment]
macOS 触控板/鼠标滚轮方向并非一定与 Windows 相反，只是**默认**相反。
可能需要给一个用户可配置选项而不是写死

**Comment by qianbinbin** [comment]
是啊，我上面已经说了

________________________________
